### DISCUSSION TITLE
Drehung um 180 Grad wird nicht berücksichtigt. Kommt sporadisch vor.

### DISCUSSION THREAD

**Discussion by TopGoalVl** [jomjol/AI-on-the-edge-device#1922] [Q&A] [answered]
Im Overview screen kommt es hin und wieder vor, dass ein jpg Image, ohne die eingestellte Drehung (bei mir 180°) angezeigt wird. Im hier dokumentierten Beispiel waren damit keine negativen Folgen verbunden. Es gibt aber auch Fälle, bei denen die fehlende Drehung zu falschen Messwerten führt.

Hat jemand ähnliche Erfahrung gemacht?
Welche Ursache kommt in Frage?

![WzDrehung180GradFehlt vom 2023-01-26 14-56-39](https://user-images.githubusercontent.com/119757556/214859312-a16693f6-6498-40c3-a1d1-d8ec4dee9829.png)

![WzDrehung180GrOk vom 2023-01-26 15-01-35](https://user-images.githubusercontent.com/119757556/214859306-1d7408fe-290f-42be-9e53-857e5934d48e.png)

**Comment by jomjol** [comment, marked as answer]
In diesem Fall ist das Aligning noch nicht so weit fortgeschritten. Im nächsten Release wird dort ein Bild kommen.
